clio-server/src/main/scala/org/broadinstitute/clio/server/service/IndexService.scala

```diff
@@ -67,4 +78,133 @@ abstract class IndexService[CI <: ClioIndex](
       elasticsearchIndex
     )
   }
+  private val keysToDrop =
+    Set(
+      ElasticsearchIndex.UpsertIdElasticsearchName,
+      ElasticsearchIndex.EntityIdElasticsearchName,
+    )
```

Comment:
hmm ok.. i thought I saw them in there but i'll double check.

Reply:
It might be because you're going around the existing query API... if they are coming through, I think `SearchService` would be the right place to strip them out based on how we've got things laid out now.